### PR TITLE
Update "pl.h".

### PR DESCRIPTION
**Pull request by J013k**
Added missing strings and updated Polish translation.

### CODE REVIEW

File: src/lang/pl.h [hunk 1]
```diff
@@ -157,20 +273,20 @@ const char *STR_PL[] = { ""
...
+    /* 46 */ , "Nast|epnie pu)s)c."
+    /* 47 */ , "Chod)zmy pop}lywa)c."
+    /* 48 */ , "Klawisz skoku i kierunk)ow@poruszaj|a mn|a pod wod|a."
+    /* 49 */ , "Ach! Powietrze! Wystarczy u*zy)c klawiszy do przodu, w lewo i w prawo@do manwerowania na powierzchni.@Naci)snij skok, aby jeszcze raz zanurkowa)c.@Albo podejd)z do kraw|edzi i naci)snij przycisk akcji, aby wyj)s)c z wody."
+    /* 50 */ , "Dobra. A teraz lepiej zdejm|e te mokre ubrania."
+    /* 51 */ , "U)smiechnij si|e!"
+    /* 52 */ , "To nic osobistego."
+    /* 53 */ , "Wci|a*z przyprawiasz mnie o b)ol g}lowy.@Przez co mam zamiar wys}la)c ci|e do diab}la!"
+    /* 54 */ , "Nie mo*zesz tak }latwo pomiata)c mn|a@i moj|a zgraj|a, Laro."
+    /* 55 */ , "Trochę p)o)zno na rozdanie nagr)od - Czy*z nie?@?Wci|a*z jednak mo*zesz wzi|a)c udzia}l."
```
Comment (XProger) [inline src/lang/pl.h:288]: ę

**Review by XProger — COMMENTED** [review]
Cool, thanks! Fix ę character please, I'll merge it.

**Comment by Haker120** [comment]
How about Polish font? I bet it'll be possible to add.

**Comment by J013k** [comment]
I am not sure if I understand you:

**Comment by elektronicznypank** [comment]
I cannot use build.sh:

clang-14: warning: treating 'c' input as 'c++' when in C++ mode, this behavior is deprecated [-Wdeprecated]
clang-14: warning: treating 'c' input as 'c++' when in C++ mode, this behavior is deprecated [-Wdeprecated]
In file included from main.cpp:15:
In file included from ../../game.h:4:
In file included from ../../core.h:795:
../../gapi/gl.h:1319:35: warning: comparison of function 'glTexImage3D' not equal to a null pointer is always true [-Wtautological-pointer-compare]
            support.tex3D       = glTexImage3D != NULL;
                                  ^~~~~~~~~~~~    ~~~~
../../gapi/gl.h:1319:35: note: prefix with the address-of operator to silence this warning
            support.tex3D       = glTexImage3D != NULL;
                                  ^
                                  &
In file included from main.cpp:15:
In file included from ../../game.h:4:
In file included from ../../core.h:812:
In file included from ../../texture.h:5:
In file included from ../../format.h:5:
In file included from ../../gameflow.h:5:
In file included from ../../lang.h:319:
../../lang/pl.h:276:195: warning: missing terminating '"' character [-Winvalid-pp-token]
    /* 43 */ , "Nie mog|e wspi|a)c si|e na g)or|e, poniewa*z szczelina jest zbyt ma}la.@Ale naci)snij w prawo, a ja b|ed|e kiwa}la si|e na boki@a*z b|edzie miejsce, a potem naci)snij do przodu.""
                                                                                                                                                                                                  ^
../../lang/pl.h:276:195: error: expected '}'
../../lang/pl.h:6:24: note: to match this '{'
const char *STR_PL[] = { ""
                       ^
2 warnings and 1 error generated.

**Comment by J013k** [comment]
Thanks for finding unnecessary `"`.

**Comment by elektronicznypank** [comment]
No problem, glad i could help.

**Pull request merged**